questionpy_sdk/webserver/question_ui/errors.py

```diff
@@ -41,10 +54,36 @@ def html_message(self) -> str:
 
 @dataclass(frozen=True)
 class RenderElementError(RenderError, ABC):
+    """Represents a generic element error which occurred during rendering."""
+
     element: etree._Element
+    template: str
+    kwargs: Mapping[str, str | Collection[str]] = field(default_factory=dict)
+
+    def _message(self, *, as_html: bool) -> str:
+        (opening, closing) = ("<code>", "</code>") if as_html else ("'", "'")
+        kwargs = {"element": f"{opening}{self.element_representation}{closing}"}
+
+        for key, values in self.kwargs.items():
+            collection = {values} if isinstance(values, str) else values
+            kwargs[key] = _comma_separated_values(collection, opening, closing)
+
+        return self.template.format(**kwargs)
```

Comment:
```suggestion
        return self.template.format_map(kwargs)
```

Und vielleicht `template_kwargs` oder `format_kwargs` statt nur `kwargs`?